### DISCUSSION TITLE
Hide Welcome Dialog

### DISCUSSION THREAD

**Discussion by camchambers** [foxglove/community#848] [Q&A] [answered]
Is there a way to disable the welcome dialog in Foxglove Studio so that it does not show each time the app is loaded?

eg. via a local storage value, url param, etc...

<img src="https://github.com/foxglove/community/assets/26011644/22ff03f7-810f-4754-8a0b-3d846ff4d666" width="400" />

**Comment by defunctzombie** [comment, marked as answer]
Nope - there's no way to disable the dialog. We use to have a "show on startup" option for it but I think that went away long ago. The welcome dialog is gonna undergo some changes soon so we'll consider this in those changes. @banisadr
